### ISSUE TITLE
Truck Groups

### ISSUE THREAD

**Issue by Paulycustomskins** [WerewolfCustoms/ModsStudio2IssueBoard#483]
Had and idea could we possibly put the truck manufactures into groups,bit like how we put accessories into groups,so instead of all the trucks being all together they would be in seperate groups.
![Screenshot 2024-05-08 085955](https://github.com/WerewolfCustoms/ModsStudio2IssueBoard/assets/72619819/72f8c5ff-f424-492c-8a2d-d8a5f12bd8d3)

**Comment by AdamLabrosse** [comment]
I suggest to modify the VehicleName entry to introduce a separator between Manufacturer and model. For example, "International Lonestar" become "International|Lonestar".

Then vehicles can appear in a list like:

**Freightliner**
| 	Scs Software - Freightliner Cascadia
**International**
|	Scs Software - International Eagle 9900i
|	Scs Software - International Lonestar

or

**Freightliner**
|	Scs Software - Cascadia
**International**
|	Scs Software - Eagle 9900i
|	Scs Software - Lonestar

All vehicles that don't have the separator will appear in a **Miscellaneous** section at the end of the list.

**Comment by Paulycustomskins** [comment]
Yeah that's what I thought,it looks better in groups,for both games it would look a lot cleaner

**Comment by Paulycustomskins** [comment]
I think the bottom would be better

**Comment by AdamLabrosse** [comment]
Here is what it looks like in MS2:
![MS2_truck_groups](https://github.com/WerewolfCustoms/ModsStudio2IssueBoard/assets/124315137/307005c5-9940-458c-b693-30a4f2231821)

**Comment by Paulycustomskins** [comment]
That looks great Adam exactly how I meant it to look

**Comment by Paulycustomskins** [comment]
Do you reckon we could do the same for the trailers in both games aswell

**Comment by AdamLabrosse** [comment]
It's already done!

**Comment by Paulycustomskins** [comment]
Nice one Adam that's gonna be look better
